proto/message_contents/public_key.proto

```diff
@@ -26,6 +27,25 @@ message UnsignedPublicKey {
     }
 }
 
+message SignedPublicKeyV2 {
+    bytes key_bytes = 1;  // embeds an UnsignedPublicKey
+    AssociationProof proof = 2; // signs key_bytes
+}
+
+message SignedPublicKeyRevocation {
+    bytes key_bytes = 1;  // embeds an UnsignedPublicKey
+    AssociationProof proof = 2; // signs key_bytes
```

Comment:
Given our talk I thought this would be a RevocationProof message? The idea being the proof's type makes it super clear what the purpose of the proof is. Otherwise, is there a distinguishing factor between an AssociationProof used to associate and one used to revoke?

Reply:
oh I must have been on a super old version of this file, disregard